src/bootstrap/setup.rs

```diff
@@ -81,6 +92,26 @@ impl fmt::Display for Profile {
     }
 }
 
+impl Step for Profile {
+    type Output = ();
+
+    fn should_run(mut run: ShouldRun<'_>) -> ShouldRun<'_> {
+        for choice in Profile::all() {
+            run = run.alias(&choice.to_string());
```

Comment:
nit: this can use as_str() instead to avoid an allocation.

Reply:
thanks! Changed, please check!